internal/test/fake_source_test.go

```diff
@@ -0,0 +1,17 @@
+package test
```

Comment:
Can you remove this file, the `Faker` structure should be enough to fake the seed, if you need you can add tests to the `faker_test.go` file